src/main/java/com/celements/cells/CellsClasses.java

```diff
@@ -196,8 +197,9 @@ BaseClass getPageDepCellConfigClass() throws XWikiException {
   BaseClass getTranslationBoxCellConfigClass() throws XWikiException {
     XWikiDocument doc;
     boolean needsUpdate = false;
-    DocumentReference translationBoxCellConfigClassRef = cellsClassConfig.getTranslationBoxCellConfigClassRef(
-        getContext().getDatabase());
+    DocumentReference translationBoxCellConfigClassRef = cellsClassConfig
```

Comment:
I do not see how this should be more readable in three lines. please keep it on two.